docs/developer_guide.md

```diff
@@ -77,7 +85,7 @@ operator later, it knows what image to use.
 ## Deploy the operator to a running Kubernetes cluster
 
 Assume you have built and pushed the Flink Operator image, then you need to have
-a running Kubernetes cluster. Verify the cluster info with
+a running Kubernetes cluster. Verify the cluster-info with
```

Comment:
I feel the original is better, because here it is natural language, the next line is the command.